dinersclub/reloadly.go

```diff
@@ -74,6 +59,43 @@ func reloadlyErrorResult(res *Result, err error, details *json.RawMessage) (*Res
 	return res, err
 }
 
+func (p *ReloadlyProvider) getCredentials(pool *pgxpool.Pool, pageid string) (*Credentials, error) {
+	query := `SELECT details FROM credentials WHERE facebook_page_id=$1 LIMIT 1`
```

Comment:
Hrm, so in the current setup, this would get you the facebook page credentials. 

What you want is something like: 

```SQL
SELECT details 
FROM credentials 
WHERE entity = 'reloadly' 
AND userid = (SELECT userid FROM credentials WHERE facebook_page_id = $1 LIMIT 1)
LIMIT 1
```